lib/assets.php

```diff
@@ -18,3 +18,18 @@
   <link rel="shortcut icon" href="<?php echo get_stylesheet_directory_uri(); ?>/favicon.ico">
   <?php
 });
+
+add_filter('wp_script_attributes', 'addScriptTypeToJs', 10, 1);
+
+function addScriptTypeToJs($attr)
```

Comment:
Nitpick: as per the above, it might be useful to include a link to the govuk-frontend changelog that describes the need for this change, so we know why we made it.